moped-database/metadata/tables.yaml

```diff
@@ -2980,8 +2983,6 @@
           - project_phase_id
           - started_by_user_id
           - subphase_id
-          - subphase_name
```

Comment:
turning off permissions for `suphase_name` - this is redundant too `moped_subphases.subphase_name`.

i will drop this column entirely in a subsequent PR.